### PR TITLE
Install tini update

### PR DESCRIPTION
**Pull request by dark0dave**
Alphine now supports tini as a package, so we can install it as one. Futher more lets be consistent with our args.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,21 +25,15 @@ VOLUME /var/jenkins_home
...
 COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-agent-port.groovy
 
 # jenkins version being bundled in this docker image
 ARG JENKINS_VERSION
 ENV JENKINS_VERSION ${JENKINS_VERSION:-2.19.4}
 
 # jenkins.war checksum, download will be validated using it
-ARG JENKINS_SHA=ea61a4ff86f0db715511d1118a4e2f0a6a0311a1
+ARG JENKINS_SHA
+ENV JENKINS_SHA ${JENKINS_SHA:-ea61a4ff86f0db715511d1118a4e2f0a6a0311a1}
```
Comment (carlossg) [inline Dockerfile]: JENKINS_SHA is only used at build time, not need for an env

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8-jdk-alpine
 
-RUN apk add --no-cache git openssh-client curl unzip bash ttf-dejavu coreutils
+RUN apk add --no-cache git openssh-client curl unzip bash ttf-dejavu coreutils tini
```
Comment (carlossg) [inline Dockerfile]: the problem is that it stops being in sync with the master branch, at a minimum it should explicitly set the same version as installed in master

File: Dockerfile (hunk 1 above)
Comment (dark0dave) [inline Dockerfile]: Good point. Please consider the tini package though.

File: Dockerfile (hunk 2 above)
Comment (dark0dave) [inline Dockerfile]: Ok, I have a set a fixed one in my latest branch. 0.9.0-r1 ok?

**Pull request closed without merging**